Sparkle/SUFileManager.m

```diff
@@ -13,11 +13,7 @@
 #include <sys/errno.h>
 #include <sys/time.h>
 
-#pragma clang diagnostic push
-// This is a private constant but it's referenced from our unit tests
-#pragma clang diagnostic ignored "-Wmissing-variable-declarations"
-NSString *SUAppleQuarantineIdentifier = @"com.apple.quarantine";
-#pragma clang diagnostic pop
+static char SUAppleQuarantineIdentifier[] = "com.apple.quarantine";
```

Comment:
Isn't the point of the warning that `AuthorizationExecuteWithPrivileges()` can alter its arguments. Meaning in order to "properly" obey the warning's complaint, you need to either stick with `const char *` and make a copy when the function is invoked, or declare this as a non-static local variable that is assigned its value immediately.